### PR TITLE
Return VOD thumbs VTT URL when available

### PR DESCRIPTION
**Pull request by mjh1**

### CODE REVIEW

File: packages/api/src/controllers/asset.ts [hunk 1]
```diff
@@ -345,6 +346,28 @@ export function getPlaybackUrl(
...
+    asset.playbackId,
+    "thumbnails",
+    "thumbnails.vtt"
+  );
+
+  let params = {
+    method: "HEAD",
+    timeout: 5 * 1000,
+  };
+  let resp = await fetchWithTimeout(vttUrl, params);
```
Comment (leszko) [inline packages/api/src/controllers/asset.ts]: Why do you need to make this HTTP call here? Can't you just return `vttUrl`?

**Review by leszko — COMMENTED** [review]
Added a comment. Also, the CI is failing, could you check?

File: packages/api/src/controllers/asset.ts (hunk 1 above)
Comment (mjh1) [reply packages/api/src/controllers/asset.ts]: Because the vtt will not always be present, it's not switched on for all users yet.

Comment (mjh1) [reply packages/api/src/controllers/asset.ts]: Tidied this up now to check the returned files on the asset

**Review by leszko — APPROVED** [review]
Looks good. One thing we could add is a unit test, somewhere here: https://github.com/livepeer/studio/blob/b5db9dc13b06d0633308344463aca62ae2349651/packages/api/src/controllers/playback.test.ts#L198

**Pull request merged**